roles/pulp_common/defaults/main.yml

```diff
@@ -63,3 +63,4 @@ pulp_pkg_undeclared_deps:
   - python3-djangorestframework-queryfields
 pulp_pkg_upgrade_all: false
 pulp_upgraded_manually: false
+pulp_disable_selinux: false
```

Comment:
With this name I expect it to set the system to `Disabled`. What about instead having:

`pulp_selinux_mode: 'enabled'` instead and then having it use that value as `state`? I think that would be clearer and more usable.

Reply:
Seems clear to me. Especially since the default is `true`.

Reply:
Added instructions in [README](https://github.com/pulp/pulp_installer/pull/346/files#diff-e5e5e1bf91103fcc1c1e75e2445b48ebR149), along with a comment in the [defaults.yml file](https://github.com/pulp/pulp_installer/pull/346/files#diff-e5e5e1bf91103fcc1c1e75e2445b48ebR149)

Reply:
Thought about this some more, and think you're right. See latest commit.